manage-apps/register.html

```diff
@@ -204,9 +206,15 @@
           $('#sandboxRegistration button[type="submit"]').attr('disabled', true);
         },
         success: function(data) {
-          console.log(data);
-          $("#sandboxRegistration").addClass('submited');
-          $('#form-success').removeClass('form-success-hidden');
+          if (data.sandboxErr !== 'No Error') {
```

Comment:
> I could manufacture a status code I guess, and pass that back

Precisely. One of the value adds that our back-end can provide is a proper HTTP interface, even in the event that CTE does not. This way, we're 1) insulating our Web client from future changes, and 2) not spreading CTE's poor API design.